client/containers/Pub/SpubHeader/SpubHeader.tsx

```diff
@@ -57,21 +56,29 @@ const SpubHeader = (props: Props) => {
 			}),
 		}).catch(() => props.updateLocalData('pub', props.pubData));
 	};
+
+	const updateSubmissionStatus = async (newSubmissionStatus: SubmissionStatus) => {
+		return apiFetch('/api/submissions', {
```

Comment:
Let me show you the wonders of `apiFetch.put()`:

```ts
apiFetch.put('/api/submissions', {
    status: newSubmissionStatus,
    id: props.pubData.submission.id,
});
```
saves you a call to `JSON.stringify()` because it does it internally :)